### ISSUE TITLE
3.5 - Participants Login As option should not be available to all users

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#10731]
**Describe the bug**
Currently you can access the Login As feature when you are logged in as a user role that would not normally have access to this feature.  Additionally for consistency the Login As option should not be available for a person who already is logged in similar to how settings user's & roles Login As function works.

Precondidtion : Using default dataset.  Can use any lower level participant of a submission.

**To Reproduce**
1. Go to VM and login the normal way with user "mfritz"
2. In the active submission list select View on submission ID 6 - Phillips — Investigating the Shared...
3. In the particpants list select the 3 dots on either participant (Daniel Barnes)
4. Observe the option table

Actual: The Login As option is available (See screenshot) 
Expected: The login option should only be available to those with access like Journal Managers, Journal Editors, or Admin

**What application are you using?**
 OJS, OMP or OPS version X.X.X

**Additional information**
Please add any screenshots, logs or other information we can use to investigate this bug report.
![Image](https://github.com/user-attachments/assets/2b4e866e-30ab-4cac-967b-1f5b5f55e8d5)

Similar to the settings if you are already logged in as a particular user and that user is a participant then the login As option should be hidden.
Example would be to normally log in as Daniel Barnes and then find a submission that they were a participant of and click the three dots.  This should be hidden similar to how it gets hidden in the settings user & roles section.
![Image](https://github.com/user-attachments/assets/12004fbe-19ee-467d-8a86-fe33a037299a)

**Comment by jardakotesovec** [comment]
@Tribunal33 Yeah, thats being worked on - https://github.com/pkp/pkp-lib/issues/10290 . Participant UI is part of the new workflow page, where every area is covered by subissue in https://github.com/pkp/pkp-lib/issues/10598 . Therefore there will be dedicated subissue for Participants once thats ready for testing.

**Comment by Tribunal33** [comment]
Added 10290 as a parent.  Thanks @jardakotesovec .

**Comment by jardakotesovec** [comment]
@Tribunal33 Created placeholder subissue https://github.com/pkp/pkp-lib/issues/10742 and used it as parent, as that issue is covering that area being finished including the frontend integration.

**Comment by jardakotesovec** [comment]
This is now available in API. Thank you @Hafsa-Naeem .

I make adjustments to use that new field. And Participant manager (parent ticket) is now ready for testing.

@Tribunal33 Closing this ticket I will leave up to you.

**Comment by Tribunal33** [comment]
This is passing now and Login As option is not appearing if the user is not able to use it.  I have added it as part of the testing for participants list #10742 

![Image](https://github.com/user-attachments/assets/b4529eed-d93e-43e4-b540-86260f4a5a80)
